### DISCUSSION TITLE
Link showing weird in chat link

### DISCUSSION THREAD

**Discussion by baharsah** [ppy/osu#16720] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/69349170/151772715-e3359d81-07e0-457f-ac61-6baff5f34119.png)

**Comment by smoogipoo** [comment, marked as answer]
This is because osu!framework doesn't yet wrap on individual characters.

**Reply by smoogipoo** [reply]
https://github.com/ppy/osu-framework/issues/5007

**Reply by Joehuu** [reply]
Also https://github.com/ppy/osu/issues/10085
